lib/dotcom_web/live/schedule_finder_live.ex

```diff
@@ -388,4 +442,140 @@ defmodule DotcomWeb.ScheduleFinderLive do
+
+              <SystemIcons.mode_icon aria-hidden line={@line_name} mode={@mode} class="shrink-0 z-20" />
+            </div>
+
+            <div class="py-2">{trip_details_header_text(upcoming_departure)}</div>
+          </div>
+          <details
+            :if={Enum.count(upcoming_departure.trip_details.stops_before) > 0}
+            class="group/details"
+          >
+            <summary class="cursor-pointer flex gap-2 items-center px-2 border-xs border-charcoal-80 border-b-0">
+              <div class="relative self-stretch w-6 shrink-0">
+                <div class={"#{route_to_class(@route)} absolute -top-[0.0625rem] left-1/2 -translate-x-1/2 w-1 z-10 h-3/4"} />
+                <div class={"#{route_to_class(@route)} absolute -bottom-[0.0625rem] left-1/2 -translate-x-1/2 w-1 z-10 h-3/4"} />
+              </div>
+              <div class="py-2">
+                {ngettext(
+                  "1 Stop Away",
+                  "%{count} Stops Away",
+                  Enum.count(upcoming_departure.trip_details.stops_before)
+                )}
+              </div>
+              <div class="shrink-0">
+                <.icon name="chevron-down" class="h-3 w-3 group-open/details:rotate-180" />
+              </div>
+            </summary>
+            <.other_stop
+              :for={other_stop <- upcoming_departure.trip_details.stops_before}
+              other_stop={other_stop}
+              route={@route}
+              stop_id={@stop_id}
+            />
+          </details>
+          <div class="[&>*:last-child_.bottom-route-line]:invisible">
+            <.other_stop
+              other_stop={upcoming_departure.trip_details.stop}
+              route={@route}
+              stop_id={@stop_id}
+            />
+            <.other_stop
+              :for={other_stop <- upcoming_departure.trip_details.stops_after}
+              other_stop={other_stop}
+              route={@route}
+              stop_id={@stop_id}
+            />
+          </div>
+        </:content>
+      </.unstyled_accordion>
+    </div>
+    """
+  end
+
+  defp other_stop(assigns) do
+    ~H"""
+    <div class="px-2 border-xs border-charcoal-80 border-b-0 flex gap-2 items-center">
+      <div class="self-stretch relative w-6 shrink-0 flex items-center justify-center">
+        <div class={"#{route_to_class(@route)} absolute -top-[0.0625rem] left-1/2 -translate-x-1/2 w-1 z-10 h-1/2 top-route-line"} />
+        <div class={"#{route_to_class(@route)} absolute -bottom-[0.0625rem] left-1/2 -translate-x-1/2 w-1 z-10 h-1/2 bottom-route-line"} />
+        <div class={"#{route_to_class(@route)} size-3.5 rounded-full border-xs border-[#00000026] z-20"} />
+      </div>
+      <div class={["py-2", @stop_id == @other_stop.stop_id && "font-bold"]}>
+        {@other_stop.stop_name}
+      </div>
+      <div class={["ml-auto", @stop_id == @other_stop.stop_id && "font-bold"]}>
+        {Timex.format!(@other_stop.time, "{h12}:{m} {AM}")}
```

Comment:
```suggestion
        {format!(@other_stop.time, :hour_12_minutes)}
```

(this is from `Dotcom.Utils.Time`, which is already imported)

Reply:
[Done](https://github.com/mbta/dotcom/pull/2819/commits/a424f192129d36177eca049b7dd35bf5ff036afa)